File: C/1329.c
#include <stdio.h>

    /*
     FELIPE DE CARVALHO ANDRADE 20222009242
     Lucas Nepomuceno Coelho 20222009055
     Este codigo foi resolvido lendo o numero de vezes que a moeda seria jogada para
     cima e depois imprimindo o valor de vezes que jhon e marry ganharam.
     */
 
int main() {
  
   int z,a,mary=0,jhon=0;
  scanf("%d",&z);
  while (z > 0){
    for (int i=0;i<z;i++){
      scanf("%d",&a);
        if(a == 0){
        mary++;
        }
      else{
        jhon++;
        }
    }
     printf("Mary won %d times and John won %d times\n",mary,jhon);
    scanf("%d",&z);
    mary =0;
    jhon=0;
  }
  
    return 0;
}

File: C#/1329.cs
using System;
using System.Globalization;

class URI {

    static void Main(string[] args) { 
        while (true) {
            int n = int.Parse(Console.ReadLine());

            if (n == 0)
                break;

            string[] line = Console.ReadLine().Split(' ');

            int maria = 0;
            int joao = 0;

            for (int i = 0; i < n; i++) {
                int v = int.Parse(line[i]);
                
                if (v == 0)
                    maria++;
                else
                    joao++;
            }

            Console.WriteLine($"Mary won {maria} times and John won {joao} times");
        }
    }
}